reanahub/reana
# helm: disable quota accounting on workflow termination

closes https://github.com/reanahub/reana/issues/628

To test:
- redeploy the cluster
- `kubectl describe deployment/reana-server | grep REANA_WORKFLOW_TERMINATION_QUOTA_UPDATE_POLICY`
should show empty string instead of `disk,cpu` as before
- run some demos and check `reana-client quota-show --resource disk` - should be 0
